config/commands.json

```diff
@@ -795,7 +795,7 @@
       "author": {
         "name": "What is JiggyTom?",
         "url": "https://en.wikipedia.org/wiki/Superhero",
-        "icon_url": "https://spee.ch/2/pinkylbryheart.png"
+        "icon_url": "pinkylbryheart.png"
```

Comment:
Shouldn't you also add https://github.com/Eniamza/lbry-wunderbot/blob/relative-path/pinkylbryheart.png here?